crates/wdk-build/src/lib.rs

```diff
@@ -755,6 +755,21 @@ impl Config {
             headers.extend(["hidpddi.h", "hidport.h", "kbdmou.h", "ntdd8042.h"]);
         }
 
+        if matches!(self.driver_config, DriverConfig::Umdf(_)) {
+            /* the hidport.h file is expected for Umdf, but located in km folder */
+            let include_directory = self.wdk_content_root.join("Include");
```

Comment:
Not something that needs to be addressed in this pr, but a lot of this logic is duplicated in a couple different places now. I think it might be good to have the `wdk_build::Config` constructor create all the "useful paths" and store them in the config struct. Possibly better is to provide public convenience methods to generate those paths. 

For useful paths, I'm thinking things that correspond with the entries defined in `WDK.props`:
![image](https://github.com/user-attachments/assets/3669555d-410e-4c6d-be64-7d91f36adcae)

Reply:
If you want to help address this in this pr, i'd just narrow to only a `kernel_mode_include_path` that corresponds with `KM_IncludePath`

Reply:
I've added a helper for this value